src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mod_tweaks/serverDataSyncer/tweakeroo/RayTraceUtilsMixin.java

```diff
@@ -20,11 +20,66 @@
 
 package me.fallenbreath.tweakermore.mixins.tweaks.mod_tweaks.serverDataSyncer.tweakeroo;
 
+//#if MC >= 12101
```

Comment:
For a completely different class implementation, please implement in a fresh source in `versions/xxx/src`, instead of `//$$` everything (which is sooo ugly)

Reply:
Like I said, Tweakeroo needs to same 1.21 treatment as MiniHUD in order for it not to crash.  I was simply following the same method that you've already used for the MiniHUD fix.